main/static/script.js

```diff
@@ -1,13 +1,17 @@
 "use strict";
 
-console.log("Hello, world!");
-
 var module = angular.module("main",[]);
 
+var chartsReady = false;
+
+var debounce = false;
```

Comment:
what's the intent you're trying to capture here? (I think I understand the code but not too sure).